jcs_sbs_sdk/service/jcs_compute_client.py

```diff
@@ -0,0 +1,341 @@
+from datetime import datetime
+from httplib import HTTPException
+from lxml import objectify
+
+from ..auth.credentials import Credentials
+from ..common import constants
+from ..common import utils
+from ..common.jcs_http_client import JCSHttpClient
+from ..model.attachment import Attachment
+from ..model.create_snapshot_request import CreateSnapshotRequest
+from ..model.create_snapshot_result import CreateSnapshotResult
+from ..model.create_volume_request import CreateVolumeRequest
+from ..model.create_volume_result import CreateVolumeResult
+from ..model.delete_snapshot_request import DeleteSnapshotRequest
+from ..model.delete_snapshot_result import DeleteSnapshotResult
+from ..model.delete_volume_request import DeleteVolumeRequest
+from ..model.delete_volume_result import DeleteVolumeResult
+from ..model.describe_snapshots_request import DescribeSnapshotsRequest
+from ..model.describe_snapshots_result import DescribeSnapshotsResult
+from ..model.describe_volumes_request import DescribeVolumesRequest
+from ..model.describe_volumes_result import DescribeVolumesResult
+from ..model.snapshot import Snapshot
+from ..model.volume import Volume
+
+
+class JCSComputeClient(JCSHttpClient):
```

Comment:
This is more of question of how our interfaces should look. Should we take objects or tokenized strings as inputs? We need to be consistent wrt other sdk's in JCS.